spec/features/consumer/shops_spec.rb

```diff
@@ -201,4 +201,3 @@ def click_link_and_ensure(link_text, check)
     end
   end
 end
-
```

Comment:
Is this to trigger a Buildkite build or Travis? It seems pretty ugly, but Buildkite has no proper search function for builds. Maybe that is something to ask them about. In Aus we tend to rebase branches.